runtime/src/tasks/qthreads/tasks-qthreads.c

```diff
@@ -315,66 +297,18 @@ void chpl_sync_destroyAux(chpl_sync_aux_t *s)
     PROFILE_INCR(profile_sync_destroyAux, 1);
 }
 
-#ifdef SUPPORT_BLOCKREPORT
-static void chapel_display_thread(qt_key_t     addr,
-                                  qthread_f    f,
-                                  void        *arg,
-                                  void        *retloc,
-                                  unsigned int thread_id,
-                                  void        *tls,
-                                  void        *callarg)
-{
-    chpl_qthread_tls_t *rep = (chpl_qthread_tls_t *)tls;
-
-    if (rep) {
-        if ((rep->lock_lineno > 0) && rep->lock_filename) {
-          fprintf(stderr, "Waiting at: %s:%zu (task %s:%zu)\n",
-                  chpl_lookupFilename(rep->lock_filename), rep->lock_lineno,
-                  chpl_lookupFilename(rep->chpl_data.task_filename),
-                  rep->chpl_data.task_lineno);
-        } else if (rep->lock_lineno == 0 && rep->lock_filename) {
-          fprintf(stderr,
-                  "Waiting for more work (line 0? file:%s) (task %s:%zu)\n",
-                  chpl_lookupFilename(rep->lock_filename),
-                  chpl_lookupFilename(rep->chpl_data.task_filename),
-                  rep->chpl_data.task_lineno);
-        } else if (rep->lock_lineno == 0) {
-          fprintf(stderr,
-                  "Waiting for dependencies (uninitialized task %s:%zu)\n",
-                  chpl_lookupFilename(rep->chpl_data.task_filename),
-                  rep->chpl_data.task_lineno);
-        }
-        fflush(stderr);
-    }
-}
-
-static void report_locked_threads(void)
-{
-    qthread_feb_callback(chapel_display_thread, NULL);
-    qthread_syncvar_callback(chapel_display_thread, NULL);
-}
-#endif  // SUPPORT_BLOCKREPORT
-
 static void SIGINT_handler(int sig)
 {
     signal(sig, SIG_IGN);
 
     if (blockreport) {
-#ifdef SUPPORT_BLOCKREPORT
-        report_locked_threads();
-#else
         fprintf(stderr,
```

Comment:
The `blockreport` and `taskreport` flags will already have been set at the time `chpl_task_init()` is called.  Would it be better to print a message there?  (Especially for `blockreport`, which the user might have set thinking it would enable deadlock detection, as it does for tasks=fifo.)

Reply:
I thought about that too, but I decided to leave it as-is because I'm not actually changing the behavior compared to master (i.e I'll I'm really doing is removing code blocks were ifdeffed out unless somebody went out of their way to modify the source)

Reply:
(And semi-related, I plan to make an issue to see how people feel about removing support for these features completely since I don't think they're very useful and they're pretty broken even under fifo -- https://github.com/chapel-lang/chapel/pull/7255, https://github.com/chapel-lang/chapel/pull/7601)

Reply:
I'm fine with that, no need for further changes on this one.  And I'll definitely have input to your new issue!